src/libraries/Common/src/Interop/Windows/Kernel32/Interop.Threading.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        internal const int WAIT_FAILED = unchecked((int)0xFFFFFFFF);
+
+        [DllImport(Libraries.Kernel32)]
+        internal static extern uint WaitForMultipleObjectsEx(uint nCount, IntPtr lpHandles, bool bWaitAll, uint dwMilliseconds, bool bAlertable);
```

Comment:
It looks like that this is trying to minimize marshaling overhead by calling `GetLastError` directly. It may make sense to also change the `bool` arguments to use `BOOL` to eliminate the rest of the marshaling.